src/commands/createFunction.ts

```diff
@@ -33,22 +33,17 @@ const requiredFunctionAppFiles: string[] = [
     path.join('.vscode', 'launch.json') // NOTE: tasks.json is not required if the user prefers to run 'func host start' from the command line
 ];
 
-function validateTemplateName(rootPath: string, name: string | undefined, language: string): string | undefined {
+function validateTemplateName(rootPath: string, name: string | undefined): string | undefined {
     if (!name) {
         return localize('azFunc.emptyTemplateNameError', 'The template name cannot be empty.');
     }
-
-    if (language === ProjectLanguage.Java) {
-        return validateJavaFunctionName(name);
-    } else {
-        if (fse.existsSync(path.join(rootPath, name))) {
-            return localize('azFunc.existingFolderError', 'A folder with the name \'{0}\' already exists.', name);
-        }
-        if (!functionNameRegex.test(name)) {
-            return localize('azFunc.functionNameInvalidError', 'Function name must start with a letter and can contain letters, digits, \'_\' and \'-\'');
-        }
-        return undefined;
+    if (fse.existsSync(path.join(rootPath, name))) {
+        return localize('azFunc.existingFolderError', 'A folder with the name \'{0}\' already exists.', name);
+    }
+    if (!functionNameRegex.test(name)) {
```

Comment:
nit: I've noticed you doing `if/if` several times when you should be doing `if/else if` like this:
```
if (...) {
    ...
} else if (...) {
    ...
}
```
Please adopt this pattern going forward. Here's a few links I found explaining why it's better:
https://stackoverflow.com/questions/9169249/why-we-use-if-else-if-instead-of-multiple-if-block-if-the-body-is-a-return-stat
https://www.codecademy.com/en/forum_questions/5595581e76b8fea07700004c